environments/hca/terraform/pubsub.tf

```diff
@@ -0,0 +1,12 @@
+module ebi_staging_notification_pubsub_topic {
+  dependencies = [module.enable_services]
```

Comment:
this hack isn't in the https://github.com/terraform-google-modules/terraform-google-pubsub/blob/master/variables.tf